medium/foundation.md

```diff
@@ -0,0 +1,16 @@
+* **io.js és a node.js Foundation**
+
+Az egyeztetés jelenlegi állapota.
+
+Egy héttel ezelőtt, Scott Hammond a Joyent CEO, meghivta egy személyes találkozóra az io.js (Technical Committee) TC csapatát, ahol feltárta elképzeléseit, miként is vezetné vissza az io.js-t a node.js projectbe.
+
+Néhány hónapon belül az io.js 23 aktiv alaptagra nőtte ki magát, illetve számos közösség épült fel. 27 különböző nyelvű csapat épült ki, és vált képessé jó minőségű release-re, valamint gyors support-ra a kiváló közösségi háttérnek köszönhetően. Képesek lettünk mindezt elérni önállóan, egy megújitott csapattal és több kontributorral projectben, mint amennyi a node.js történetében valaha is volt.
+
+Csupán jobbá teheti az io.js-t ha pihenni hagyjuk a jövőbeni kérdéseket a node.js-szel kapcsolatos szétvélást illetően. Készek vagyunk magunk mögött hagyni mindezt, de nem akarjuk feláldozni az elért fejlődést valamint a jelenlegi szabályainkat és önálló működésünket amit ezidáig kialaktottunk.
+
+Aktiv beszélgetések folynak a Joyent-tel és ha egyszer a Foundation szakértői irányitás felé tendál, akkor értesülni fogtok erről a Github-on keresztül, hogy az io.js-nek érdemes-e csatlakozni ehhez a kezdeményezéshez. Ennek az iránynak az elfogadásában mindenki kiveheti a szerepét, mivel egy nyilt szavazással lesz eldöntve a következő lépés egy publikus TC meeting során.
```

Comment:
*Aktiv beszélgetések folynak a Joyent-tel és ha egyszer a Foundation szakértői irányitás felé tendál, akkor értesülni fogtok erről a Github-on keresztül, hogy az io.js-nek érdemes-e csatlakozni ehhez a kezdeményezéshez.* -> *Jelenleg is folynak a tárgyalások a Joyent-tel. Amint a Foundation kialakítja elképzeléseit a project vezetéséről, a csatlakozásról szóló issue-ban meg fogjuk vitatni, hogy érdemes-e csatlakozni hozzá (a vita természetesen a korábbi irányelveink mentén fog történni)*
*Ennek az iránynak az elfogadásában mindenki* -> *A döntésben mintenki*